substrate-sgx/externalities/Cargo.toml

```diff
@@ -1,19 +1,30 @@
 [package]
 name = "sgx-externalities"
-version = "0.3.0"
+version = "0.4.0"
 authors = ["Integritee AG <hello@integritee.network> and Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
+
+# no_std
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive", "chain-error"]}
 derive_more = "0.99.16"
-environmental = { version = "1.0.1", default-features = false}
-sgx_tstd      = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git", features = ["untrusted_fs","net","backtrace"]}
-sgx_types     = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git"}
-sgx_serialize = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
-sgx_serialize_derive = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
-sgx_log       = { package = "log", git = "https://github.com/mesalock-linux/log-sgx"}
+environmental = { version = "1.0.1", default-features = false }
+log = { version = "0.4", default-features = false }
+serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
+serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
+
+# sgx dependencies
+sgx_tstd      = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git", features = ["untrusted_fs","net","backtrace"], optional = true }
+sgx_types     = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
 
 [features]
 default = ["std"]
-std = []
\ No newline at end of file
+std = [
+    "log/std",
+    "serde/std",
+    "serde_json/std",
+]
```

Comment:
Please enable std, of all crates in std. Missing crates may have unforeseen effects.

Reply:
That's what I've done here, no? At least for the ones I've added

Reply:
Thanks for pointing this out @clangenb .

@mullefel could you also add the ones you have not added yourself? That'd include `environmental` and `codec`. Thanks alot!